internal/templates/containercapabilities/template.go

```diff
@@ -0,0 +1,218 @@
+package containercapabilities
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"golang.stackrox.io/kube-linter/internal/check"
+	"golang.stackrox.io/kube-linter/internal/diagnostic"
+	"golang.stackrox.io/kube-linter/internal/matcher"
+	"golang.stackrox.io/kube-linter/internal/objectkinds"
+	"golang.stackrox.io/kube-linter/internal/templates"
+	"golang.stackrox.io/kube-linter/internal/templates/containercapabilities/internal/params"
+	"golang.stackrox.io/kube-linter/internal/templates/util"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	reservedCapabilitiesAll             = "all"
+	matchLiteralReservedCapabilitiesAll = "(?i)" + reservedCapabilitiesAll
+)
+
+var (
+	literalReservedCapabilitiesAllMatcher func(string) bool
```

Comment:
This can just be initialized statically:
```go
	literalReservedCapabilitiesAllMatcher = func() func(string) bool {
               m, err := matcher.ForString(matchLiteralReservedCapabilitiesAll) 
               utils.Must(err)
               return m
        }()

```